infrastructure/tooling/src/main.js

```diff
@@ -147,6 +147,7 @@ program.command('dev:init')
 
 program.command('dev:db:upgrade')
   .description('Run `yarn db:upgrade` for a development environment')
+  .option('--db-version <v>', 'Downgrade to this DB version (optional, defaults to latest)')
```

Comment:
Wouldn't this defeat the purpose of having a dev:db:downgrade?

Reply:
Whoops, that was copy-paste -- should be 
```suggestion
  .option('--db-version <v>', 'Upgrade to this DB version (optional, defaults to latest)')
```